packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -135,7 +137,24 @@ export type InjectedWalletMetadata = BaseWalletMetadata & {
   downloadUrl: string;
 };
 
-export type InjectedWalletBehaviour = BaseWalletBehaviour;
+export interface AccountImportData {
+  accountId: string;
+  privateKey: string;
+}
+
+export interface AccountImportSecureContextParams {
+  accounts: Array<AccountImportData>;
+}
+
+export type InjectedWalletBehaviour = Modify<
+  BaseWalletBehaviour,
+  {
+    buildImportAccountsUrl?: never;
```

Comment:
Can we rethink about applying `buildImportAccountsUrl` as never on `InjectedWallet` ?
Currently Meteor wallet is defined as injected wallet on `wallet-selector` but their approach towards to wallet migration behaves just like browser wallet. (So based on our flow, Meteor wallet should be using `buildImportAccountsUrl`.)

Furthermore, some wallet may support more than one type of interface, (eg. practically, wallet like Meteor wallet, does have both browser AND extension wallet where they can support both `buildImportAccountsUrl` and `importAccountsInSecureContext` interfaces)

Reply:
> Currently Meteor wallet is defined as injected wallet on wallet-selector but their approach towards to wallet migration behaves just like browser wallet.

Hmm do we know why they're configured as an injected wallet if they function as a browser wallet? I don't think it's worth making exceptions for misconfigured wallets but if there's something we've missed here then it's important to incorporate that into our model.

It is extremely risky to have an injected wallet send a link, as injected wallets use unencrypted data but anything in a URL must be encrypted. We need to make this as straightforward and secure as possible, so intermixing wallet types and making one-off exceptions breaks our proposed implementation.

> Furthermore, some wallet may support more than one type of interface, (eg. practically, wallet like Meteor wallet, does have both browser AND extension wallet where they can support both buildImportAccountsUrl and importAccountsInSecureContext interfaces)

This is a limitation of `wallet-selector` AFAIK. Since each wallet only has exactly one type, each package represents a wallet + mode (of which most wallets will only have one). I don't think this is a relationship we want to change though; if I'm selecting Meteor as my target wallet then isn't it important that I select whether I want the mobile wallet or browser wallet? I'd imagine they have quite different implementation considerations.

So at the moment I would prefer to have wallets implement a separate package for a different mode, assuming that there is not significant overlap between the same wallet in different modes. If this ends up being untenable for wallet builders then we have some potentially major architecture decisions to make.